predicates/backups.go

```diff
@@ -0,0 +1,44 @@
+/*
+Copyright 2023.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package predicates
+
+import (
+	"sigs.k8s.io/controller-runtime/pkg/event"
+	"sigs.k8s.io/controller-runtime/pkg/predicate"
+)
+
+var backupLabels = []string{
+	"velero.io/backup-name",
+	"velero.io/restore-name",
+}
+
+// IgnoreBackups implements a default create predicate function to ignore all create events triggered but backup tools.
+type IgnoreBackups struct {
+	predicate.Funcs
+}
+
+// Create returns false if the object associated with the create event does not contain any of the
```

Comment:
Shouldn't this be the other way around, `Create returns false if the object associated with the create event contains any of the expected backup labels. It will return true otherwise.` ?

Reply:
Fixed.